monkey/infection_monkey/monkey.py

```diff
@@ -59,6 +61,10 @@ def initialize(self):
         self._parent = self._opts.parent
         self._default_tunnel = self._opts.tunnel
         self._default_server = self._opts.server
+        try:
+            self._default_server_port = self._default_server.split(':')[1]
+        except KeyError:
+            self._default_server_port = ''
```

Comment:
You'll have a broken URI in this case.
it'll be 
IP: 

Which is invalid

Reply:
good catch.